install/Knative-with-ICP.md

```diff
@@ -0,0 +1,173 @@
+# Knative Install on IBM Cloud Private
+
+This guide walks you through the installation of the latest version of
+[Knative Serving](https://github.com/knative/serving) and [Knative Build](https://github.com/knative/build) using pre-built images and
+demonstrates creating and deploying an image of a sample `hello world` app onto
+the newly created Knative cluster on [IBM Cloud Private](https://www.ibm.com/cloud/private).
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires a v3.1.1 standard [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster. Before you can install Knative, you must first complete all the steps that are provided in the [IBM Cloud Private standard cluster installation instructions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html). For example:
+
+1. Install Docker for your boot node only
+
+2. Set up the installation environment
+
+3. Customize your cluster
+
+4. Set up Docker for your cluster nodes
+
+5. Deploy the environment
+
+6. Verify the status of your installation
+
+## Installing Istio
+
+[Follow the instructions to install and run Istio in IBM Cloud Private](https://istio.io/docs/setup/kubernetes/quick-start-ibm/#ibm-cloud-private).
+
+## Installing Knative components
+
+You can install the Knative Serving and Build components together, or individually.
+
+### Installing Knative Serving and Build components
+
+Run the following command to deploy [Knative Serving](https://github.com/knative/serving) and [Knative Build](https://github.com/knative/build)
+
+```shell
+curl -L https://github.com/knative/serving/releases/download/v0.2.2/release-lite.yaml \
+  | sed 's/LoadBalancer/NodePort/' \
+  | kubectl apply --filename -
+```
+
+### Installing Knative Serving only
+
+Replace `release-lite.yaml` to `serving.yaml` file, and then run the following command:
+
+```shell
+curl -L https://github.com/knative/serving/releases/download/v0.2.2/serving.yaml \
+  | sed 's/LoadBalancer/NodePort/' \
+  | kubectl apply --filename -
+```
+
+### Installing Knative Build only
+
+Replace `release-lite.yaml` to `build.yaml` file, and then run the following command:
+
+```shell
+curl -L https://github.com/knative/serving/releases/download/v0.2.2/build.yaml \
+  | sed 's/LoadBalancer/NodePort/' \
+  | kubectl apply --filename -
+```
+
+### Update the image security policy
+You need to update the [image security policy](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/manage_images/image_security.html) to allow the cluster to pull the Knative image when you enable the `image-security-enforcement` at the time of [IBM Cloud Private](https://www.ibm.com/cloud/private) installation.
```

Comment:
This says "at the time of installation" & "to pull the Knative image", both of which make it sound like this security policy needs to be created before you run the "kubctl apply" commands to install Knative.

Does the whole "Update the image security policy" section need to move up before the "Installing Knative components" steps?

Reply:
Yes, you are right. The section **Update the image security policy** and **Update pod security policy** are all need to move up before the "Installing Knative components" steps. I put this two section after "install IBM Cloud Private"